gulpfile.js

```diff
@@ -108,7 +112,8 @@ gulp.task('build', [
   'build/long-stack-trace-zone.js',
   'build/long-stack-trace-zone.min.js',
   'build/wtf.js',
-  'build/wtf.min.js'
+  'build/wtf.min.js',
+  'build/async-test.js'
 ]);
```

Comment:
I don't know the full details but just looking at the other specs - Do we also need a async-test.min.js?